src/lib/libdwarf/CMakeLists.txt

```diff
@@ -100,10 +104,14 @@ foreach(i RANGE ${targetCount})
 	add_library(${target} ${type} ${SOURCES} ${HEADERS}
             ${GENNAMES_OUTPUT} ${CONFIGURATION_FILES})
 
+    add_library(libdwarf::${target} ALIAS ${target})
+    
 	set_folder(${target} src/lib/libdwarf)
     target_compile_options(${target} PRIVATE ${COMPILER_FLAGS}
             ${DW_FWALL})
 	msvc_posix(${target})
+    
+    target_compile_definitions(${target} PUBLIC ${DEFS})
```

Comment:
LIBDWARF_STATIC, LIBDWARF_BUILD, and PIC need to be public definitions so projects using libdwarf gets the right dllimport/dllexport/visibility annotations.